ui/src/components/layout/Main.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import {Box} from "@mui/material";
+import Header from "../Header";
+import Container from "@mui/material/Container";
+import {UserSpace} from "../UserSpace";
+import {Route, Routes} from "react-router-dom";
+import {About, Creation, Home} from "../../pages";
+import {styled} from "@mui/material/styles";
+import DrawerComponent from "../Drawer/Drawer";
+import Help from "../../pages/Help";
+
+function MainLayout() {
+  const [open, setOpen] = React.useState(false);
+
+  return (
+    <Container maxWidth="lg">
+      <BoxStyled>
+        <Header handleDrawerOpen={() => setOpen(true)}/>
+        <>
+          <ContainerStyled fixed>
+            <UserSpace>
+              <Routes>
```

Comment:
Any way we can reuse the `ROUTES` from `src/constants.ts` here? I'm afraid that having the routes duplicated makes it more difficult to maintain.

Reply:
It's a good idea, I thought to take it to the separate story as it's preferable and more customizable to use createBrowserRouter method so that I will refactor it all together 
https://reactrouter.com/en/main/routers/create-browser-router

Reply:
ok created an issue for this https://github.com/ChainSafe/Multix/issues/108